var/spack/repos/builtin/packages/grads/package.py

```diff
@@ -74,6 +74,16 @@ def patch(self):
             if self.spec.satisfies("^g2c@1.8.0:"):
                 filter_file("G2_VERSION", "G2C_VERSION", "src/gacfg.c")
 
+    # Can use newer versions of HDF5, but 1.10 is the last API GrADS supports
```

Comment:
nit while I am here, can you move the comment be next to where the flags are defined?